src/AppBundle/Entity/Answer.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: lukas
+ * Date: 16.11.13
+ * Time: 22.30
+ */
+
+namespace AppBundle\Entity;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="answers")
+ */
+class Answer
+{
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\ManyToOne(targetEntity="questions")
+     */
+    private $questionId;
+    /**
+     * @ORM\Column(length=500)
+     */
+    private $text;
```

Comment:
No type for field text provided, also no need to provide length if you choose type="text"